package.json

```diff
@@ -28,6 +28,7 @@
         "packages/*"
     ],
     "dependencies": {
+        "@massalabs/as-transformer": "^0.2.2-dev.20230728112508",
```

Comment:
do we really want the commit id here?

Reply:
No, but for now we don't have a choice, I guess we should create an issue to update the dependency when release is made.